src/lib/dev-environment/dev-environment-lando.js

```diff
@@ -98,6 +98,25 @@ async function getLandoConfig() {
 
 const appMap: Map< string, App > = new Map();
 
+async function getApplication( lando: Lando, instancePath: string ): Promise< App > {
```

Comment:
Could we somehow make the difference between getApplication and getLandoApplication more clear? What does the `getLandoApplication` actually do?

Reply:
I have renamed one of them.

Initially, there used to be only `getLandoApplication()`. It ensured that we initialize every application only once. Then we added a feature to recover broken landofiles. The common part was extracted into `getApplication()` (which is now `initLandoApplication()`).